pkg/v1/tkg/test/tkgctl/shared/e2e_common.go

```diff
@@ -223,3 +269,69 @@ func E2ECommonSpec(ctx context.Context, inputGetter func() E2ECommonSpecInput) {
 		By("Test successful !")
 	})
 }
+
+func getNextAvailableTkrVersion(ctx context.Context, currentTkrVersion string, mngClient client.Client) (string, error) {
+	tkrList := runv1alpha3.TanzuKubernetesReleaseList{}
+	mngClient.List(ctx, &tkrList)
```

Comment:
Would it be possible to check the return value (non nilness of ```err```) of this List call?

Reply:
stale.